mpf/config_players/plugin_player.py

```diff
@@ -50,30 +53,36 @@ def register_player_events(self, config, mode=None, priority=0):
 
         self.bcp_client = self._get_bcp_client(config)
 
+        m = self.machine
```

Comment:
We increased max-line-length to 120 in the landscape/prospector config (https://github.com/missionpinball/mpf/blob/dev/.landscape.yml). It should ist this one if you run prospector from the git root.

For this you could also break the lines into a multi-line statement instead.